controller/search_blackbox_test.go

```diff
@@ -675,6 +687,21 @@ func (s *searchBlackBoxTest) TestSearchQueryScenarioDriven() {
 		require.Len(s.T(), result.Data, 3+5)
 	})
 
+	s.T().Run("space=ID AND (state=resolved OR iteration=sprint2) using EQ", func(t *testing.T) {
+		filter := fmt.Sprintf(`
+			{"$AND": [
+				{"space":"%s"},
```

Comment:
Well, I have only used `$EQ` in the leaf node, not in the middle. I will change it.